src/components/Navigation.jsx

```diff
@@ -2,50 +2,47 @@ import React from 'react';
 import _ from 'lodash';
 import { Link } from 'react-router';
 
-export default class Navigation extends React.Component {
+export default function Navigation(props) {
+  // Render all categories in nav bar
+  let categories = ["off-campus", "on-campus", "commentary", "creative", "in-focus"];
+  let renderCategories =
+    _.map((categories || []), function(category) {
+      return(
+        <li key={category} className="navigation__categories__item">
+          <Link to={"/category/"+category} activeClassName="navigation__categories__item--active">
+            {category}
+          </Link>
+        </li>
+      )
+    });
 
-  render() {
-    // Render all categories in nav bar
-    let categories = ["off-campus", "on-campus", "commentary", "creative", "in-focus"];
-    let renderCategories =
-      _.map((categories || []), function(category) {
-        return(
-          <li key={category} className="navigation__categories__item">
-            <Link to={"/category/"+category} activeClassName="navigation__categories__item--active">
-              {category}
-            </Link>
-          </li>
-        )
-      });
-
-    return (
-      <div>
-        <div className="header">
-          <div className="header__search">
-            <input
-              className="header__search__main"
-              type="text"
-              placeholder="Search The Gazelle"
-            />
-            <div className="header__search__tail" />
-          </div>
-          <div className="header__title">
-            <Link to="/" className="header__title--content">{this.props.appName}</Link>
-          </div>
-          {/* TODO: change link to archives list */}
-          <Link to="/" className="header__issueId">{"Issue 55"}</Link>
+  return (
+    <div>
+      <div className="header">
+        <div className="header__search">
```

Comment:
Yes, _generally_ it tends to be on the right. On the other hand, plenty of big brands have it on the left including Facebook, 8tracks, Pinterest, and Gmail. Wikipedia had a great article on why they moved their search bar from the left to the right of their page. [Here's the article](https://blog.wikimedia.org/2010/06/15/usability-why-did-we-move-the-search-box/) if you're interested. On the other hand, [another study](https://www.uie.com/brainsparks/2006/03/22/does-search-need-to-be-in-the-upper-right/) found that it's best to have it in the top left. More or less, it seems, it really doesn't seem to matter so long as 1) it's at the top of the page and 2) it's not in a sidebar. 

The reasoning behind having it in the top left on The Gazelle is because the top right is _generally_ used navigational purposes. It is more of a suggestion to the audience to explore previous issues in their intended form rather than to search for an article. The hope is that users understand that ISSUE X is a tool for navigation rather than simply a declaration and use it as such.